presto-hudi/src/main/java/com/facebook/presto/hudi/HudiSplitManager.java

```diff
@@ -120,7 +129,8 @@ public ConnectorSplitSource getSplits(
             Path partitionPath = new Path(hudiPartition.getStorage().getLocation());
             String relativePartitionPath = FSUtils.getRelativePartitionPath(tablePath, partitionPath);
             fsView.getLatestFileSlicesBeforeOrOn(relativePartitionPath, timestamp, false)
-                    .map(fileSlice -> createHudiSplit(table, fileSlice, timestamp, hudiPartition))
+                    .map(fileSlice -> createHudiSplit(
+                            table, fileSlice, timestamp, hudiPartition, splitWeightProvider))
```

Comment:
nit: one argument per line

Reply:
Given that the line is not long enough, I put them into one line with the lambda function.